### PR TITLE
Quick can now take multiple files as input

### PR DESCRIPTION
**Pull request by jacobstanley**
Usage:

```
mafia quick -i../disorder-core/src $(find src -name '*.hs')
```

^ I think with a bit of work we can make `mafia repl` do this automatically and at the same time prevent the source dependencies we know we will have loaded (in this case `disorder-core`) from being built.

/cc @markhibberd

### CODE REVIEW

File: main/mafia.hs [hunk 1]
```diff
@@ -234,37 +234,41 @@ bench args = do
...
-            <> [ path ]
+  return $ [ "-no-user-package-db" ]
+        <> (fmap ("-i" <>)           includes)
+        <> (fmap ("-package-db=" <>) databases)
+        <> paths
+
+checkEntryPoint :: File -> EitherT MafiaError IO ()
+checkEntryPoint file = do
+  exists <- doesFileExist file
+  case exists of
```
Comment (charleso) [inline main/mafia.hs]: `unless` (or `unlessM` if you have it)? :) /nit

**Comment by markhibberd** [comment]
For the non-bash non-savages:

```
mafia quick -i../disorder-core/src **/*.hs
```

File: main/mafia.hs (hunk 1 above)
Comment (jacobstanley) [inline main/mafia.hs]: oh yeah, `unlessM` is in P now I think

**Comment by jacobstanley** [comment]
> For the non-bash non-savages:
> 
> ```
> mafia quick -i../disorder-core/src **/*.hs
> ```

wow, TIL

**Comment by jacobstanley** [comment]
@charleso does this look ok now?

**Comment by charleso** [comment]
:+1:

**Pull request merged**